src/resources/Correction/BenefitCompany.ts

```diff
@@ -2,7 +2,7 @@ import { CorrectionTypes, FilingCodes, NameRequestTypes } from '@/enums/'
 import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module/'
 import { ResourceIF } from '@/interfaces/'
 
-export const BenefitCompanyStatementResource: ResourceIF = {
+export const BenCorrectionResource: ResourceIF = {
```

Comment:
Eventually we should rename all the resource objects to prevent conflicts and confusion.

Should I do it now?